src/writer/fail_on_skipped.rs

```diff
@@ -86,17 +95,24 @@ where
                     f,
                     Feature::Rule(
                         r,
-                        Rule::Scenario(sc, Scenario::Step(st, Step::Skipped)),
+                        Rule::Scenario(sc, Scenario::Background(st, Step::Skipped)),
                     ),
-                ) => map_failed(f, Some(r), sc, st),
+                ) => map_failed_bg(f, Some(r), sc, st),
```

Comment:
Actually, we want to map our failed `Background` `Step`s into `event::Scenario::Background`, not into `event::Scenario::Step` like with regular `Step`s